### PR TITLE
[Auth restructure 2] Some fixes for `rspotify-model` and `rspotify-macros`

### PR DESCRIPTION
**Pull request by marioortizmanero**
Part 2 of the #207 split-up.

This:
* Fixes some small things in `rspotify-model`
* Fixes some small things in `rspotify-macros`

These things are needed later to test the crate without cyclical dependencies, and mostly clean up a few things. Both of the crates I modified should compile without problems, but not the main one.

### CODE REVIEW

**Comment by ramsayleung** [comment]
It seems, some CI checks failed, it will be automatically fixed after merged part 3?

**Comment by marioortizmanero** [comment]
Yes. The CI will only work after merging the third PR

**Comment by ramsayleung** [comment]
Beside this nitpick I mention, everything looks good to me.

File: rspotify-macros/Cargo.toml [hunk 1]
```diff
@@ -13,4 +16,3 @@ edition = "2018"
 [dev-dependencies]
 serde_json = "1.0.57"
 chrono = { version = "0.4.13", features = ["serde", "rustc-serialize"] }
```
[the remark this answers is not in the document]
Comment (marioortizmanero) [reply rspotify-macros/Cargo.toml:18]: These are dev dependencies, they're only needed for tests. You're right that we don't need chrono, though. I've removed it.

**Review by ramsayleung — APPROVED** [review]

**Pull request merged**